docs/topics_5/rn-known-issues.adoc

```diff
@@ -3,9 +3,9 @@
 [id='rn-known-issues_{context}']
 = Known issues
 
-At the time of release, the following known issues are identified as major issues in {ProductName} {DocInfoProductNumber}.
+At the time of release, the following known issues have been identified as major issues worth highlighting.
 
-// For a full list of all known issues, see the link:https://issues.redhat.com/issues/?filter=12356604[open {ProductShortName} issues] in Jira.
+For a full list of all known issues, see the list of link:https://issues.redhat.com/browse/WINDUPRULE-607?filter=12357834[{DocInfoProductNumber} known issues] in Jira.
```

Comment:
I thought Phil wanted the text to read "5.1.1" issues. If so, change this attribute.